CONTRIBUTING.md

```diff
@@ -0,0 +1,166 @@
+
+1.  Identify an existing issue you would like to work on, or submit an issue describing your proposed change to the repo in question.
+1.  The repo owners will respond to your issue promptly.
+1.  Fork the desired repo, develop and test your code changes.
+1.  Submit a pull request.
+
+## How to run the Scorecard Monitor locally
+
+Currently, this project is consumed as a GitHub Action, so local development is quite limited. In order to test the full workflow, you can consume directly your fork and branch from another project or by adding a new workflow in your fork.
+
+Aside from this, it is possible to test certain things locally, like the `utils.js` file. Just check the test folder `_tests_/utils.test.js` to get a better idea.
+
+## How to install the dependencies
+
+First, check that you are using Node v20+ and then execute `npm ci` instead of `npm i` or `npm install` as you want to mimic the pipeline steps in order to avoid discrepancies later on with the `dist/` as the dependencies are included there.
+
+## How to run the tests
+
+Currently, the project is using [Jest](https://jestjs.io/) and [Snapshot Testing](https://jestjs.io/docs/snapshot-testing).
+
+You have several options to run the tests:
+- `npm run test`: this will run the tests
+- `npm run test:update`: this will run the tests and update the snapshots
+- `npm run test:coverage` this will run the tests and generate a coverage report as terminal output and in HTML format that can be found in the `coverage/` folder
+- `npm run test:watch`: this will run the tests when you make changes in any of the project's files.
+
+## How to run the linter
+
+This project uses [JavaScript Standard Style](https://standardjs.com/). If you are not familiar with this style, you can make your changes and run `npm run lint:fix` when you are ready. The linter will fix most of the issues for you and it will highlight any additional issue that requires manual work.
+
+To check that your files are properly linted, you can run `npm run lint`. This review won't make changes to your files.
+
+## PR Process
+
+Every PR should be annotated with an icon indicating whether it's a:
+
+-   Breaking change: :warning: (`:warning:`)
+-   Non-breaking feature: :sparkles: (`:sparkles:`)
+-   Patch fix: :bug: (`:bug:`)
+-   Documentation changes (user or developer): :book: (`:book:`)
+-   Infra/Tests/Other: :seedling: (`:seedling:`)
+-   No release note: :ghost: (`:ghost:`)
+
+Use :ghost: (no release note) only for the PRs that change or revert unreleased
+changes, which don't deserve a release note. Please don't abuse it.
+
+Prefer using the `:xyz:` aliases over the equivalent emoji directly when possible.
+
+Individual commits should not be tagged separately, but will generally be
+assumed to match the PR. For instance, if you have a bugfix in with a breaking
+change, it's generally encouraged to submit the bugfix separately, but if you must put them in one PR, you should mark the whole PR as breaking.
```

Comment:
Let's drop this section for now.
I have... feelings about all of the emoji usage and we should probably think of a way to simplify the release notes generation process.

Reply:
I agree.. I prefer to use tags :)

Reply:
Done in https://github.com/ossf/scorecard-monitor/pull/85/commits/9b7101a563b40753f5d84e29c961b80469ca1a25